src/content-scripts/annotations/AnnotationLocator.ts

```diff
@@ -3,15 +3,15 @@ import 'rangy/lib/rangy-classapplier';
 import 'rangy/lib/rangy-highlighter';
 import 'rangy/lib/rangy-textrange';
 import 'rangy/lib/rangy-serializer';
-import { LocatedAnnotation } from '../../common/store/tabs/tab/annotations/types';
+import { LocatedAnnotation } from 'common/store/tabs/tab/annotations/types';
 import { annotationRootNode } from '../settings';
 import { Range as XPathRange } from 'xpath-range';
-import { escapeRegExp } from 'tslint/lib/utils';
```

Comment:
@kaglowka I found out that importing from typescript or tslint is very costly. It leads to slowness of build and some webpack warnings  when source-map-support does not know how to handle _special_ files from typescript which normally are used to compiling not being compiled.
It's not a big deal for us, because we actually do not use that typescript code that trans-piles TS into JS, it was just a replaceable util in this case, but still enough to triggers import of massive codebase ;)

Reply:
Good job! The compilation time is actually a great deal, with how much TS compilation takes along with extension reloading etc.

Reply:
I just realized I imported it from tslint by accident :)